### DISCUSSION TITLE
Weekly stock data

### DISCUSSION THREAD

**Discussion by amitchouksey** [alvarobartt/investpy#413] [Q&A] [answered]
Hello, thanks for creating this package. Appreciate all your efforts!!
Currently function `get_stock_historical_data` pulls the daily data. I was just wondering if it is possible for you to enhance it to pull daily/weekly/monthly data.

**Comment by alvarobartt** [comment, marked as answer]
Hi @amitchouksey thanks for the kind words! :hugs:

It can be done indeed, you just need to set the `interval='weekly'` parameter, which can also be `monthly` and `daily` which is the default value.

For example:

```python
import investpy

data = investpy.get_stock_historical_data(stock='AAPL', country='united states', from_date='01/01/2020', to_date='01/01/2021', interval='weekly')
```

For more information check https://investpy.readthedocs.io/_api/stocks.html#investpy.stocks.get_stock_historical_data

**Reply by amitchouksey** [reply]
Thank you very much!! This is awesome!!
